zerver/lib/upload.py

```diff
@@ -139,12 +143,14 @@ def upload_image_to_s3(
 
 def get_file_info(request, user_file):
     # type: (HttpRequest, File) -> Tuple[str, str]
-    uploaded_file_name = user_file.name
+    uploaded_file_name = user_file.name.encode('ascii')
```

Comment:
why do we need to encode as ascii here?  Is it more for unusual cases where a user uploads a file not using jquery-filedrop?